lib/feedparser/index.js

```diff
@@ -71,6 +71,7 @@ function FeedParser (options) {
   if (!('strict' in this.options)) this.options.strict = false;
   if (!('normalize' in this.options)) this.options.normalize = true;
   if (!('addmeta' in this.options)) this.options.addmeta = true;
+  if (!('parseDate' in this.options)) this.options.parseDate = true;
```

Comment:
We've already parsed the date string into a `Date` object. With this option enabled, we'll actually be stringifying the date. So, let's rename this option: maybe `stringifydate`. (If you have a different suggestion, that's ok, but let's also ensure the option names are consistent -- one word, all lowercase.) (I know, I know, there's `resume_saxerror` which has an underscore -- 🤷‍♂️ )

Also, we need to preserve the current behavior as the default, so if the option is not provided, it's value should be `false`.